sunpy/map/sources/sdo.py

```diff
@@ -43,15 +42,23 @@ def processing_level(self):
 
     def _get_mpl_normalizer(self):
         """Returns a Normalize object to be used with AIA data"""
-        # byte-scaled images have most likely already been scaled
-        if self.data.dtype == np.uint8:
-            return None
-
-        mean = self.mean()
-        std = self.std()
-
-        vmin = max(0, mean - 3 * std)
-        vmax = min(self.max(), mean + 3 * std)
+        vmin = self.data.min()
+        vmax = self.data.max()
+
+        if self.wavelength.value == 304:
```

Comment:
The selection of arbitrary values makes me nervous.

Also, I think this would be better implemented as a dict which is indexed with the wavelength, especially if it was exposed in a user accessible way (somehow, maybe).